shoop/themes/classic_gray/static_src/js/custom.js

```diff
@@ -16,6 +16,19 @@ function showPreview(productId) {
     });
 }
 
+function setProductListViewMode(isInListMode) {
+    if (typeof(Storage) !== "undefined") {
+        localStorage.setItem("product_list_view_list_mode", (isInListMode) ? "list" : "grid");
```

Comment:
Wrap the whole ternary operator expression in parentheses, not just the first operand.